src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-mount-details/cephfs-mount-details.component.html

```diff
@@ -0,0 +1,51 @@
+<cd-modal (hide)="cancel()">
+  <ng-container class="modal-title">
+    <span i18n>Attach commands</span>
+  </ng-container>
+  <ng-container class="modal-content">
+    <div class="modal-body">
+      <h5 class="fw-bold"
+          i18n>
+        Using Mount command
+      </h5>
+      <prep [ngClass]="['hstack', 'gap-3','text-wrap', 'p-4', 'mb-4', 'code-block']">
+        <code id="mount">
+          <span>sudo &lt;client_user&gt;&#64;{{ mountData?.fsId }}.{{ mountData?.fsName }}={{ mountData?.rootPath }} &lt;mount/point&gt;</span>
+        </code>
+        <cd-copy-2-clipboard-button source="mount"></cd-copy-2-clipboard-button>
+      </prep>
```

Comment:
we have a [code-block](https://github.com/ceph/ceph/blob/main/src/pybind/mgr/dashboard/frontend/src/app/shared/components/code-block/code-block.component.html). can we reuse that here?

Reply:
you can improve it if you want

Reply:
done